### ISSUE TITLE
[Request] Swipe function on the smartphone

### ISSUE THREAD

**Issue by DickenSerm** [glanceapp/glance#128]
I think it would be really cool if you could switch between columns in the mobile view by swiping left and right. 

Would that be possible without a lot of effort?  

I really like the dashboard, I'm excited to see what new widgets will appear in the future :)

**Comment by svilenmarkov** [comment]
Hey, thanks for the idea!

I think it would be pretty nice and intuitive being able to swipe between columns. In terms of implementation, I'm not entirely sure how difficult it would be as I haven't played around with any of the `touch*` events so far and something tells me there will be edge cases I'll need to account for. I have a bunch of things planned for the next release so I'll leave this one for v0.7.0

**Comment by dhanadhan** [comment]
It could be helpful to implement swipe gestures for navigating between feeds within a group. For example, if I have five subreddits displayed on a single tile, allowing left and right swipes to cycle through them would be convenient. Currently, it is difficult to navigate/click from the top of the screen. Perhaps swiping in the middle of the screen could move between groups, while swiping near the bottom could switch between columns.

**Comment by jasii** [comment]
> It could be helpful to implement swipe gestures for navigating between feeds within a group. For example, if I have five subreddits displayed on a single tile, allowing left and right swipes to cycle through them would be convenient. Currently, it is difficult to navigate/click from the top of the screen. Perhaps swiping in the middle of the screen could move between groups, while swiping near the bottom could switch between columns.

Yes, I was just thinking this today after I added a bunch of RSS feeds in groups to my page. I view Glance on my phone multiple times a day and it would be really nice to swipe between them. Right now I currently have to swipe the title over and then select the new title tab. It's tedious once you have too many titles to fit horizontally on your phone.
